packages/devtools-reps/src/reps/reps.css

```diff
@@ -243,7 +243,7 @@
 
 button.open-accessibility-inspector,
 button.open-inspector {
-  mask: url(/images/open-inspector.svg) no-repeat;
+  mask: url("chrome://devtools/skin/images/vview-open-inspector.png") no-repeat;
```

Comment:
This is required to show the "open in inspector" icon; even the console appears broken in Nightly.

Reply:
How's that? I just tested on latest Nightly and it seems fine in the console: 

<img width="507" alt="image" src="https://user-images.githubusercontent.com/578107/49677970-9ee57b80-fa4f-11e8-89c2-505ddaa700ad.png">

Would you have the STR for the broken console?

Reply:
My bad; the `./firefox` directory hash was behind and it wasn't showing the icon.  I updated and we're good.  Sorry!